services/NotificationAgent/src/notification_agent/seen.clj

```diff
@@ -26,14 +20,14 @@
   (let [uuids (:uuids (parse-body body))]
     (validate-uuids uuids body)
     (db/mark-notifications-seen user uuids)
-    (successful-seen-response user)))
+    user))
```

Comment:
`successful-seen-response` was doing more than just wrapping its argument, so it should probably be restored or inlined.